lab-gurpreet/Readme

```diff
@@ -0,0 +1,8 @@
+as a developer
+     need to install using npm install node
```

Comment:
Nice, so this is starting to look a little better. You don't need to tell your users to run 'npm init' or specify which packages to install. Really, the only thing you need to tell the user here is to run 'npm install'. You don't have to specify the packages - npm install will by default install all of the dependencies listed in the package.json file.